mlrun/serving/server.py

```diff
@@ -306,18 +306,25 @@ def v2_serving_init(context, namespace=None):
     if not data:
         raise MLRunInvalidArgumentError("failed to find spec env var")
     spec = json.loads(data)
+    context.logger.info("Initializing server from spec")
     server = GraphServer.from_dict(spec)
     if config.log_level.lower() == "debug":
         server.verbose = True
     if hasattr(context, "trigger"):
         server.http_trigger = getattr(context.trigger, "kind", "http") == "http"
+    context.logger.info_with(
+        "Setting current function",
+        current_function=os.environ.get("SERVING_CURRENT_FUNCTION", ""),
+    )
     server.set_current_function(os.environ.get("SERVING_CURRENT_FUNCTION", ""))
+    context.logger.info("Initializing states")
     server.init_states(context, namespace or get_caller_globals())
+    context.logger.info("Initializing graph steps")
     serving_handler = server.init_object(namespace or get_caller_globals())
```

Comment:
i would suggest to also dive into `init_states` `init_object` and see if logs can be applied to them as well 
(as long as they r executed once, it is ok)